Apps/WebClient/src/NewClientApp/src/components/ReleaseNoteComponent.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { computed, useSlots } from "vue";
+
+interface Props {
+    date: string;
+    version: string;
+    title: string;
+}
+defineProps<Props>();
+
+const slots = useSlots();
+const hasSlot = computed<boolean>(() => {
+    return slots.default !== undefined;
+});
+</script>
+
+<template>
+    <div>
+        <v-row>
+            <v-col class="text-body-1 font-weight-bold">
+                {{ date }} – {{ version }} – {{ title }}
+            </v-col>
+        </v-row>
+        <v-row v-if="hasSlot" class="mt-1 mb-2">
+            <v-col class="px-3 py-0">
```

Comment:
This doesn't affect my review, but if you only want to push the alignment one direction I would bother applying padding to both sides, so `pl-3` would have been fine.

Reply:
n/a anymore